src/Symfony/Component/PropertyAccess/PropertyAccessor.php

```diff
@@ -345,6 +345,11 @@ private function readPropertiesUntil(array $zval, PropertyPathInterface $propert
             }
 
             $propertyValues[] = $zval;
+
+            // We are null safe and the current value is null, no need to continue.
```

Comment:
this paraphrases the code, it can be removed ;)

Reply:
Done :)